Automattic/jetpack
# Fix/videopress update metadata warnings

Fixes 1158-gh-Automattic/greenhouse

#### Changes proposed in this Pull Request:

* add input checks to prevent php errors in logs

Note: Considered using null-coalescing operator `??` to keep the amount of new code low, but it doesn't appear to be used anywhere else in the codebase (potentially due to PHP version requirements?)

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?


nope

#### Testing instructions:






* apply PR
* run `jetpack docker tail`
* edit a videopress media item in Calypso (should work fine with wordpress.com prod)
* change 1 field and wait for it to sync (check jetpack site's media library directly to ensure it synced `/wp-admin/upload.php`)
* no warning logs should have been written to the log
* change each field one at a time, and ensure they all eventually sync over to Jetpack, and nothing is "accidentally" cleared out or replaced with a `null`
